### PR TITLE
Updated the readme with the logo

### PR DESCRIPTION
**Pull request by awartoft**
Just noticed we don't have the zf logo in the zf2 repo readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-### Welcome to the *Zend Framework 2.3* Release!
+![Logo](http://framework.zend.com/images/logos/ZendFramework-logo.png)
```
Comment (Ocramius) [inline README.md]: Is the logo available in the repo?

Comment (gianarb) [inline README.md]: @Ocramius this is a good idea? In ZF1 is an external url (same url).. it isn't in the repo

Comment (froschdesign) [inline README.md]: > In ZF1 is an external url (same url).. it isn't in the repo

Also in [zf-web](/zendframework/zf-web) and [zf1-extras](/zendframework/zf1-extras).

Comment (Ocramius) [inline README.md]: @froschdesign should be fixed imo, but isn't a big deal. I just want to do it right if we introduce it.

Comment (gianarb) [inline README.md]: I don't add it in the repo

**Comment by Ocramius** [comment]
Merged, thanks!

`master`: zendframework/zf2@67694e53c4e1b60f8bbd4af3a7e2b6e7532d9bad
`develop`: zendframework/zf2@ff0b5a4337d16af0c4602b4e05de8047b0ac4b8c

**Pull request merged**